pkg/reconciler/ksvc/resources/knative_configuration.go

```diff
@@ -7,23 +7,15 @@ import (
 	knservingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
 	"github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha1"
 	"github.com/kubeflow/kfserving/pkg/constants"
-	"github.com/kubeflow/kfserving/pkg/frameworks/custom"
-	"github.com/kubeflow/kfserving/pkg/frameworks/tensorflow"
+	"github.com/kubeflow/kfserving/pkg/frameworks"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func CreateModelServingContainer(modelName string, modelSpec *v1alpha1.ModelSpec) *v1.Container {
-	if modelSpec == nil {
-		return nil
-	} else if modelSpec.Tensorflow != nil {
-		return tensorflow.CreateTensorflowContainer(modelName, modelSpec.Tensorflow)
-	} else if modelSpec.Custom != nil {
-		return custom.CreateCustomContainer(modelSpec.Custom)
-	} else {
-		//TODO(@yuzisun) handle other model types
-		return &v1.Container{}
-	}
+	// ignoring error response since we assume validation ensured the modelSpec is valid
```

Comment:
Bad to swallow errors. Assumptions may not always be true over time and we should fail fast.. Throw a fatal "unexpected" or something of the sort.